source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
@eslutsky This should be a paragraph in Chapter 3 for users that upgrade from 4.2.